src/test/java/be/syntra/devshop/DevshopBack/testutilities/JsonUtils.java

```diff
@@ -0,0 +1,28 @@
+package be.syntra.devshop.DevshopBack.testutilities;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public final class JsonUtils {
+    private ObjectMapper mapper;
+
+    @Autowired
+    public JsonUtils(ObjectMapper mapper) {
+        this.mapper = mapper;
+    }
+
+
+    public String asJsonString(final Object object) {
+        try {
+            return this.mapper.registerModule(new JavaTimeModule())
+                    .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+                    .writeValueAsString(object);
```

Comment:
normally i would expect setting the properties of the mapper outside of this method, but that will only be needed if this class gets a second method that relies on the same settings.